packaging/osx/package-osx.sh

```diff
@@ -28,7 +28,7 @@ fi
 PACKAGE_DIR=$REPOROOT/artifacts/packages/pkg
 [ -d "$PACKAGE_DIR" ] || mkdir -p $PACKAGE_DIR
 
-PACKAGE_NAME=$PACKAGE_DIR/dotnet-cli-x64.${DOTNET_CLI_VERSION}.pkg
+PACKAGE_NAME=$PACKAGE_DIR/dotnet-osx-x64.${DOTNET_CLI_VERSION}.pkg
```

Comment:
While you are in here, would you mind changing all the places in the script where this name is duplicated? That way we don't have this issue going forward?  I see two places below where this name is copied.